chronograf.go

```diff
@@ -9,35 +9,36 @@ import (
-	ErrInvalidCellOptionsText          = Error("invalid text wrapping option. Valid wrappings are 'truncate', 'wrap', and 'single line'")
-	ErrInvalidCellOptionsSort          = Error("cell options sortby cannot be empty'")
-	ErrInvalidCellOptionsColumns       = Error("cell options columns cannot be empty'")
+	ErrUpstreamTimeout                      = Error("request to backend timed out")
+	ErrSourceNotFound                       = Error("source not found")
+	ErrServerNotFound                       = Error("server not found")
+	ErrLayoutNotFound                       = Error("layout not found")
+	ErrDashboardNotFound                    = Error("dashboard not found")
+	ErrUserNotFound                         = Error("user not found")
+	ErrLayoutInvalid                        = Error("layout is invalid")
+	ErrDashboardInvalid                     = Error("dashboard is invalid")
+	ErrSourceInvalid                        = Error("source is invalid")
+	ErrServerInvalid                        = Error("server is invalid")
+	ErrAlertNotFound                        = Error("alert not found")
+	ErrAuthentication                       = Error("user not authenticated")
+	ErrUninitialized                        = Error("client uninitialized. Call Open() method")
+	ErrInvalidAxis                          = Error("Unexpected axis in cell. Valid axes are 'x', 'y', and 'y2'")
+	ErrInvalidColorType                     = Error("Invalid color type. Valid color types are 'min', 'max', 'threshold', 'text', and 'background'")
+	ErrInvalidColor                         = Error("Invalid color. Accepted color format is #RRGGBB")
+	ErrInvalidLegend                        = Error("Invalid legend. Both type and orientation must be set")
+	ErrInvalidLegendType                    = Error("Invalid legend type. Valid legend type is 'static'")
+	ErrInvalidLegendOrient                  = Error("Invalid orientation type. Valid orientation types are 'top', 'bottom', 'right', 'left'")
+	ErrUserAlreadyExists                    = Error("user already exists")
+	ErrOrganizationNotFound                 = Error("organization not found")
+	ErrMappingNotFound                      = Error("mapping not found")
+	ErrOrganizationAlreadyExists            = Error("organization already exists")
+	ErrCannotDeleteDefaultOrganization      = Error("cannot delete default organization")
+	ErrConfigNotFound                       = Error("cannot find configuration")
+	ErrAnnotationNotFound                   = Error("annotation not found")
+	ErrInvalidCellOptionsText               = Error("invalid text wrapping option. Valid wrappings are 'truncate', 'wrap', and 'single line'")
+	ErrInvalidCellOptionsSort               = Error("cell options sortby cannot be empty'")
+	ErrInvalidCellOptionsColumns            = Error("cell options columns cannot be empty'")
+	ErrOrganizationConfigFindOrCreateFailed = Error("failed to find or create organization config")
```

Comment:
What's this error for?

Reply:
failure state when FindOrCreate fails -- used by `noop` and in tests

Reply:
I think I'd split it into separate errors like

`OrganicationConfigNotFound` and custom errors for creation